### DISCUSSION TITLE
v7 beta is out!

### DISCUSSION THREAD

**Discussion by katspaugh** [katspaugh/wavesurfer.js#2789] [Announcements]
After a solid two months of work, I'm excited to announce version 7.0.0-beta of Wavesurfer and its plugins. I've basically rebuilt it from scratch! Check it out here: https://github.com/katspaugh/wavesurfer.js/tree/beta

What's new, you ask?

* It's faster (zoom zoom!)
* It's smaller (less is more, right?)
* And guess what? It's all in TypeScript!

But wait, there's more! We've got two shiny new plugins: [Envelope](https://wavesurfer-js.org/examples/#envelope.js) and [Record](https://wavesurfer-js.org/examples/#record.js).

Also, we've jazzed up our [homepage](https://wavesurfer-js.org). It's all about live examples now, plus we've got a new logo!

Big shout-out to everyone who took the alpha for a spin and reported bugs. With over 40 bug reports and suggestions, you've all helped make this new version even better.

<img width="1396" alt="Screenshot 2023-05-27 at 09 44 37" src="https://github.com/wavesurfer-js/wavesurfer.js/assets/381895/c9bbc487-6771-4754-8558-5259f0c475da">

**Comment by TwanLuttik** [comment]
As I was using this for my project, I was very excited to hear that a new version has come out, and I'm already looking forward to using the new version of this. Keep up the great work 🔥

**Comment by marcelblum** [comment]
Going to be some time before I get a chance to fully update to using this version but - awesome!!
